core/drivers/stm32_gpio.c

```diff
@@ -1312,6 +1326,7 @@ static TEE_Result apply_sec_cfg(void)
 				     'A' + bank->bank_id);
 				STAILQ_REMOVE(&bank_list, bank, stm32_gpio_bank,
 					      link);
+				free(bank);
```

Comment:
Could you also mention this fix in the commit message body?

Reply:
Don't forget to update the commit message.

Reply:
Yes, done, thanks for the remidner